components/cluster-coordinator/org.wso2.carbon.cluster.coordinator/org.wso2.carbon.cluster.coordinator.rdbms/src/main/java/org/wso2/carbon/cluster/coordinator/rdbms/RDBMSCommunicationBusContextImpl.java

```diff
@@ -693,7 +777,7 @@ public void clearMembershipEvents() throws ClusterCoordinationException {
                 boolean isCoordinatorNode = coordinatorNodeId.equals(nodeId);
                 if (resultSet.getBlob(3) != null) {
                     int blobLength = (int) resultSet.getBlob(3).length();
-                    byte[] bytes = resultSet.getBlob(3).getBytes(0L, blobLength);
+                    byte[] bytes = resultSet.getBlob(3).getBytes(1L, blobLength);
```

Comment:
Why 1L?

Reply:
This was a bug fix. The first position is 1 and not 0. Also getBytes() expects a long value. Hence the 1L

Reply:
Ack.